src/workers.jl

```diff
@@ -120,7 +120,23 @@ function trigger_profile(w::Worker, timeout_profile_wait, from::Symbol=:manual)
         elseif Sys.isbsd()
             kill(w.process, 29)  # SIGINFO
         end
-        sleep(timeout_profile_wait) # Leave time for it to print the profile.
+        sleep(timeout_profile_wait) # Leave time to output the profile.
+    end
+    return nothing
+end
+
+# Spawn GDB to request thread backtraces followed by invoking jl_print_task_backtraces
+# to get task backtraces. We wait for GDB to finish so no sleep is needed. Thread backtraces
+# are written to `gdb.btall`, which is overwritten if present.
+function trigger_backtraces(w::Worker, from::Symbol=:manual)
+    if Sys.islinux()
```

Comment:
Which part of this is Linux specific? 
- On Mac could we have an `lldb` call that achieves the same thing as `gdb`? 
- does BSD support `gdb`, so this could be `islinux() || isbsd()`?

Can you add the answer to the PR description please, so if this feature is later requested for other OS we can look back at this PR?

Reply:
> We could support FreeBSD/OpenBSD since they have gdb, but I don't have a machine that would let me test. Note that Sys.isbsd() returns true for Mac.

There is `Sys.isopenbsd()` which is what we'd want. But i take your point about not being able to test it 👍

Reply:
opened https://github.com/JuliaTesting/ReTestItems.jl/issues/158